shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -269,8 +344,47 @@ - (BOOL)isEqualTo:(FlutterTextRange*)other {
 }
 @end
 
+// A FlutterTextInputView that masquerades as a UITextField, and forwards
+// selectors it can't respond to to a shared UITextField instance.
+//
+// Relevant API docs claim that password autofill supports any custom view
+// that adopts the UITextInput protocol, automatic strong password seems to
+// currently only support UITextFields, and password saving only supports
+// UITextFields and UITextViews, as of iOS 13.5.
+@interface FlutterSecureTextInputView : FlutterTextInputView
+@property(class, nonatomic, assign, readonly) UITextField* textField;
+@end
+
+@implementation FlutterSecureTextInputView
++ (UITextField*)textField {
+  static UITextField* _textField = nil;
+  if (_textField == nil) {
+    _textField = [[UITextField alloc] init];
```

Comment:
Why not have a UITextField per FlutterSecureTextInputView?  This seems a bit risky considering we are forwarding unknown messages all to a singular instance.

Reply:
The unrecognized selectors sent to `FlutterSecureTextInputView` that I'm seeing in my demo app:
```
rightView
leftView
textAlignment
font
_setContentCoverViewMode:
_setContentCoverView:
_setBackgroundCoverViewMode:
_setBackgroundCoverView:
_layoutManager
```

Reply:
I think the assumption here is that the `UITextField` instance is just a stateless (for the states that matter to us) selector sponge. For the selectors I'm seeing the hack seems to work fine. If this hack doesn't work then we should probably switch to a completely different approach (e.g., subclassing `UITextField`).  
If there are other selectors that may set the `UITextField` to a broken state, then having separate instances probably won't prevent that from happening? If we make each `FlutterSecureTextInputView` use its own `UITextField`, we are going to reuse the `UITextField` anyway when we reuse the `FlutterSecureTextInputView`.

Reply:
It isn't stateless though since we are forwarding messages to it that could be setting all kinds of state.
```objc
[inputView1 setAccessibilityHint:@"hey"];
print(inputView2.accessibilityHint);
```
for example

Reply:
Updated to create a `UITextField` per instance. What I meant was if any of the states those unrecognized selectors can change matter to us or iOS itself, then using this hack is pretty much a lost cause.